lib/icalendar.ex

```diff
@@ -38,13 +38,15 @@ defmodule ICalendar do
 end
 
 defimpl ICalendar.Serialize, for: ICalendar do
-  def to_ics(calendar) do
+  def to_ics(calendar, options \\ []) do
     events = Enum.map(calendar.events, &ICalendar.Serialize.to_ics/1)
+    vendor = Keyword.get(options, :vendor, "ICalendar")
```

Comment:
Would "Elixir ICalendar" be more precise?

Reply:
Indeed. :+1:

Reply:
Done.